### PR TITLE
[backend] sqlc and logger setup

### LINKED ISSUE #31: Create DB querier
Initialize `sqlc` and implement DB querier for further implementation.

### PR DESCRIPTION
**Pull request by Error0229**
**Implementation of `sqlc` Environment**

**How to Use:**
1. Install `sqlc` using the following command:
   ```bash
   go install github.com/sqlc-dev/sqlc/cmd/sqlc@latest
   ```
2. Set up the PostgreSQL environment using Docker:
   ```bash
   docker-compose up postgres
   ```
3. Run `migrate up` to initialize the database schema. Don't forget to [export](https://stackoverflow.com/a/30969768/12890363) the database's environment variables.

4. Add SQL queries file in the `db/queries` directory. After that, execute `sqlc generate`. The corresponding functions will be generated in the `db` folder.

---

**Additional Changes:**
- Introduce a test folder for development testing:
  ```bash
  go run ./test/.
  ```
  Use this command to execute development tests.

- Change logger for further [usage](https://github.com/uber-go/zap).

---

### CODE REVIEW

**Comment by x06lan** [comment]
we can remove dummy_test.go now
https://github.com/jykuo-love-shiritori/twp/blob/57380cfe20a49f81ab971e479e6580542c5af648/dummy_test.go#L1-L9

**Comment by NOOBDY** [comment]
it can be left in for future reference

File: pkg/router/buyer.go [hunk 1]
```diff
@@ -13,8 +15,11 @@ import (
 // @Success 200
 // @Failure 401
 // @Router /buyer/order [get]
-func buyerGetOrderHistory(c echo.Context) error {
-	return c.NoContent(http.StatusOK)
+func buyerGetOrderHistory(c *db.DB, logger *zap.SugaredLogger) echo.HandlerFunc {
+	return func(c echo.Context) error {
+
+		return c.NoContent(http.StatusOK)
+	}
```
Comment (x06lan) [inline pkg/router/buyer.go]: should unified the parameter name to db or another

Comment (Error0229) [reply pkg/router/buyer.go]: fixed

**Review by x06lan — APPROVED** [review]

**Review by NOOBDY — APPROVED** [review]
LGTM

**Pull request merged**